.github/actions/spelling/excludes.txt

```diff
@@ -54,14 +54,26 @@ SUMS$
 \.zip$
 ^consolegit2gitfilters\.json$
 ^dep/
-^oss/
+^doc/reference/master-sequence-list.csv$
 ^doc/reference/UTF8-torture-test\.txt$
+^oss/
+^src/host/ft_uia/run\.bat$
+^src/host/runft\.bat$
+^src/host/runut\.bat$
 ^src/interactivity/onecore/BgfxEngine\.
 ^src/renderer/wddmcon/WddmConRenderer\.
+^src/terminal/adapter/ut_adapter/run\.bat$
+^src/terminal/parser/delfuzzpayload\.bat$
+^src/terminal/parser/ft_fuzzer/run\.bat$
 ^src/terminal/parser/ft_fuzzer/VTCommandFuzzer\.cpp$
-^src/types/ut_types/UtilsTests.cpp$
+^src/terminal/parser/ft_fuzzwrapper/run\.bat$
+^src/terminal/parser/ut_parser/run\.bat$
+^src/tools/integrity/packageuwp/ConsoleUWP\.appxSources$
+^src/tools/lnkd/lnkd\.bat$
+^src/tools/pixels/pixels\.bat$
+^src/tools/texttests/fira\.txt$
```

Comment:
* Warning: src/terminal/parser/ft_fuzzwrapper/run.bat: line 1, columns 1-1, Warning - Skipping `src/terminal/parser/ft_fuzzwrapper/run.bat` because there seems to be more noise (5) than unique words (1) (total: 5 / 1). (noisy-file)
* Warning: src/terminal/parser/ut_parser/run.bat: line 1, columns 1-1, Warning - Skipping `src/terminal/parser/ut_parser/run.bat` because there seems to be more noise (4) than unique words (2) (total: 4 / 2). (noisy-file)
* Warning: src/tools/integrity/packageuwp/ConsoleUWP.appxSources: line 1, columns 1-1, Warning - Skipping `src/tools/integrity/packageuwp/ConsoleUWP.appxSources` because there seems to be more noise (10) than unique words (8) (total: 25 / 26). (noisy-file)
* Warning: src/tools/lnkd/lnkd.bat: line 1, columns 1-1, Warning - Skipping `src/tools/lnkd/lnkd.bat` because there seems to be more noise (3) than unique words (0) (total: 3 / 0). (noisy-file)
* Warning: src/tools/pixels/pixels.bat: line 1, columns 1-1, Warning - Skipping `src/tools/pixels/pixels.bat` because there seems to be more noise (3) than unique words (0) (total: 3 / 0). (noisy-file)
* Warning: src/tools/texttests/fira.txt: line 1, columns 1-1, Warning - Skipping `src/tools/texttests/fira.txt` because there seems to be more noise (2) than unique words (0) (total: 2 / 0). (noisy-file)